lib/commands/dist-tag.js

```diff
@@ -190,8 +190,8 @@ class DistTag extends BaseCommand {
     }
   }
 
-  async fetchTags (spec, opts) {
-    const data = await regFetch.json(
+  static async fetchTags (spec, opts) {
```

Comment:
This doesn't appear to be needed now that we're pulling full packuments in publish.